README.md

```diff
@@ -56,17 +56,21 @@ Later, named scenes represent customer-specific applications or demos that we su
 
 At this time, each scene is tightly coupled to a specific robot configuration. Future expansion is planned to decouple these elements, enabling the execution of all scenes with all major robotic arm models that Viam currently supports.
```

Comment:
once motion planning with constraints is merged we should add corresponding tests.
suggestion: file ticket.

Reply:
- [ ] Add ticket for new motion tests